server/index.js

```diff
@@ -2,186 +2,62 @@ const express = require('express');
-//         newUser.name  = profile.name.givenName + ' ' + profile.name.familyName; // we will need to look at the passport user profile to see how names are returned. This was the suggestion in the article though
-//         newUser.email = profile.emails[0].value; // facebook can return multiple emails so we'll take the first
-//
-//         // save our user to the database
-//         newUser.save(function(err) {
-//           if (err) {
-//             throw err;
-//           }
-//           // if successful, return the new user
-//           return done(null, newUser);
-//         });
-//       }
-//     });*/
-//   }
-// ));
-
-// passport.use(new Strategy({
-//     clientID: config.FACEBOOK_clientID,
-//     clientSecret: config.FACEBOOK_clientSecret,
-//     callbackURL: 'http://localhost:3000/auth/facebook/callback'
-//   },
-//   function(accessToken, refreshToken, profile, done) {
-//     // User's Facebook profile is supplied as the user record
-//     process.nextTick(function() {
-
-//     User.findOne({ 'facebookId' : profile.id }, function(err, user) {
-
-//     // if there is an error, stop everything and return that
-//     // ie an error connecting to the database
-//     if (err) {
-//       return done(err);
-//     }
-
-//     if (user) {
-//       return done(null, user); // user found, return that user
-//     } else {
-//       // if there is no user found with that facebook id, create them
-//         var newUser            = new User();
-//         // set all of the facebook information in our user model
-//         newUser.facebookId  = profile.id; // set the users facebook id                   
-//         newUser.favRecipes = {}; //we set the user's favorite recipes to a blank object
-//         newUser.week_one = [{}, {}, {}, {}, {}, {}, {}]; // I thought we'd start with a blank object for each day. Then, if a user adds a recipe to a meal for a day we'll begin adding new key-value pairs here
-//         newUser.week_two = [{}, {}, {}, {}, {}, {}, {}];
-//         newUser.name  = profile.name.givenName + ' ' + profile.name.familyName; // we will need to look at the passport user profile to see how names are returned. This was the suggestion in the article though
-//         newUser.email = profile.emails[0].value; // facebook can return multiple emails so we'll take the first
-        
-//         // save our user to the database
-//         newUser.save(function(err) {
-//             if (err)
-//                 throw err;
-
-//             // if successful, return the new user
-//             return done(null, newUser);
-//         });
-//       }
-//     });
-//   })
-// }
-
-/*
-
-Thoughts on Meal Add/Remove Flow:
-
--User is going to login 
--Immediately upon login, we run a function that populates state for the user with their latest values from the database
--> assuming a completely new user, the week state for them would look as follows:
- 
-this.state = {
```

Comment:
@nealfwilliams Looks like you're removing a significant amount of code. Is this supposed to be removed? Where are the newUser properties being set?